crawler/src/main/java/com/brandwatch/minibcr/crawler/crawlerjob/Crawler.java

```diff
@@ -0,0 +1,21 @@
+package com.brandwatch.minibcr.crawler.crawlerjob;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import com.brandwatch.minibcr.common.domain.Resource;
+import com.brandwatch.minibcr.crawler.services.CrawlerService;
+
+@Component
+public class Crawler {
+
+    @Autowired
+    CrawlerService crawlerService;
+
+    @Scheduled(fixedDelay = 5000)
+    public void getResources() {
+        Resource resource = crawlerService.fetchTweet();
+        System.out.println(resource);
```

Comment:
is this temporary? Otherwise you should use a logger or add a TODO to change it later.

Reply:
Yes, this is really temporary. Next PR will save resources to kafka topics so no logging would be required.